bc_obps/registration/tests/endpoints/test_reportingactivity_endpoints.py

```diff
@@ -0,0 +1,46 @@
+import pytest
```

Comment:
I know I'm being tedious here, but would recommend renaming this file to `test_reporting_activities_endpoints` so that the name matches the API endpoint more closely